addons/ace_interact/fnc_radioListChildrenActions.sqf

```diff
@@ -42,12 +42,18 @@ private _radioList = [] call EFUNC(api,getCurrentRadioList);
     };
 
     private _currentChannel = [_x] call EFUNC(api,getRadioChannel);
+    if (/* TODO: max channels == 1*/ false) then {
```

Comment:
Example code for that:
`_channelsCount = count ([_radioId, "getState", "channels"] call EFUNC(sys_data,dataEvent)) - 1;`

Source: https://github.com/IDI-Systems/acre2/blob/master/addons/sys_prc343/radio/fnc_setCurrentChannel.sqf#L25

Reply:
Thanks!

Reply:
That returns a large number still, looks like AN/PRC-77 counts band channels?